products/BellHybrid/services/db/agents/ShuffleQuoteModel.cpp

```diff
@@ -28,6 +28,14 @@ namespace
         }
         return (std::abs(current - last) > dayInSec); // If the day is the same but different month/year
     }
+
+    bool isTimeElapsed(std::time_t current, std::time_t last, std::chrono::minutes minutes)
+    {
+        const auto currentTime = system_clock::from_time_t(current);
```

Comment:
Do we need to convert timestamps to that intermediate type and `duration_cast` to minutes? If I get the idea of this method right IMO it will be easier to just do `std::abs(current - last) / utils::time::secondsInMinute`

Reply:
You're right, it will be easier. Done.